blog/model.go

```diff
@@ -46,3 +47,11 @@ type BlogLikes struct {
 	ID        string   `bson:"_id" json:"_id,omitempty"`
 	UsersList []string `bson:"users_list" json:"users_list"`
 }
+
+
+type MailLists struct {
+	ID              primitive.ObjectID 		`bson:"_id,omitempty" json:".id,omitempty"`
+	Email			string					`bson:"email" json:"email"`
+	Subscribed		bool					`bson:"subscribed" json:"subscribed"`
```

Comment:
By default, everyone on the mailing list is subscribed. Do you think we should still keep record of people that unsubscribed? 🤔 . If we should then this is fine, means we will never delete records, we just toggle the subscribed field to show their interest. If we shouldn't then we do not need the subscribed field. We will just get rid of the doc from the collection. Let me know what you think.

Reply:
Went with that flow...cox of the team's plan to always have a form of record...like never cause any permanent deletion

Reply:
You have the final decision though....

We still keep their record after they unsubscribe or not